### DISCUSSION TITLE
As a site visitor, I miss an explanation of how Aesara is different

### DISCUSSION THREAD

**Discussion by pkese** [aesara-devs/aesara#966] [Q&A] [answered]
I wish there was a sentence somewhere near the top of README explaining why use Aesara instead of something else.

Something akin to *"unlike Tensorflow and PyTorch, Aesara lets you XXXXX, and compared to JAX, Aesara does YYYYY."*

And if it is based on Theano (I'm just guessing), then maybe explain what's was so special about Theano, as to be worth keeping it around.

This would be of great help for anyone trying to understand and get familiar with this project.

Oh, and as an end-user, I'm interested more in how my experience will be different with Aesara, rather than if it is implemented in Python or C++.

**Comment by brandonwillard** [comment, marked as answer]
See https://github.com/aesara-devs/aesara/discussions/879.

**Reply by pkese** [reply]
Yes. This is wonderful. I love it.

**Reply by ricardoV94** [reply]
Should we add it to the docs?

**Reply by brandonwillard** [reply]
Something like that, yeah.
